chia/wallet/wallet_user_store.py

```diff
@@ -65,12 +65,15 @@ async def create_wallet(
                 (id, name, wallet_type, data),
             )
             await cursor.close()
+            wallet = await self.get_last_wallet()
+            if wallet is None:
+                raise ValueError("Failed to get the just-created wallet")
```

Comment:
If any of the previous SQL commands fail, will they also throw `ValueError` ?

Reply:
I don't know offhand and it probably depends on how they fail.